### DISCUSSION TITLE
Why ids are used with implodes in queries?

### DISCUSSION THREAD

**Discussion by bradparks** [chevereto/chevereto#46] [Q&A] [answered]
![image](https://github.com/chevereto/chevereto/assets/185477/44c92e70-9b77-4132-b1f1-0235d49c9a3b)

Hey... I'm wondering why you're using sprintf, implodes and concatting strings here to build sql queries?
if any of the ids that are being used in these queries were user supplied, can't they be a sql injection?

I see at times you're using parameters and I get that the table names are under your control, but the ids I find surprising? 

Thanks for your time :D !

**Comment by rodber** [comment, marked as answer]
This codebase is filled with a ridiculous amount of ways to do things as it is a reflection of my experimentation during the first leg of my career.

I did many wacky things back in the day, but it works!

**Comment by bradparks** [comment]
thanks for the response - I totally get that - there are always bugs, for sure, so no worries :D, and I'm not sure there's any issue there, but I wanted to just ask, and appreciate the response!

**Comment by bradparks** [comment]
you've done a ton of work here, so thanks 👍 great project, and I haven't done php for ~ 10 years, but it looks really solid and has lots of integrations, which is awesome. I bought a 1 year subscription, and will be a voice in the forums, but thanks for your effort, and your wife's effort in doing this - hope you're doing well 😄 1 
92L93927E1323504E
